Cargo.toml

```diff
@@ -12,7 +12,7 @@ edition = "2021"
 api-server = { path = "src/api_server" }
 anyhow = "1.0"
 async-trait = "0.1.31"
-attestation-service = { git = "https://github.com/confidential-containers/attestation-service.git" }
+attestation-service = { git = "https://github.com/confidential-containers/attestation-service.git" , default-features = false, features =[ "rvps-server", "in-toto" ] }
```

Comment:
Maybe we should also enable `"all-verifier"` feature of attestation-service here?

Reply:
I changed that part now. The default cargo build and make kbs are unmodified, and added a new `kbs-no-verifier` makefile target.